shiny/ui/_html_deps_external.py

```diff
@@ -17,16 +20,49 @@
 
 
 def bootstrap_deps() -> list[HTMLDependency]:
-    dep = HTMLDependency(
-        name="bootstrap",
+    bootstrap_source: HTMLDependencySource = {
+        "package": "shiny",
+        "subdir": "www/shared/bootstrap/",
+    }
+
+    bootstrap_css = HTMLDependency(
+        name="bootstrap-css",
```

Comment:
I would consider pulling the names out into variables so that it's more obvious that `bootstrap_deps()` and `bootstrap_deps_suppress()` have shared logic